src/validators/unusable_trip.rs

```diff
@@ -0,0 +1,30 @@
+use gtfs_structures::Trip;
+
+use crate::issues::Issue;
+use crate::{IssueType, Severity};
+
+pub fn validate(gtfs: &gtfs_structures::Gtfs) -> Vec<Issue> {
+    gtfs.trips
+        .values()
+        .filter_map(|trip| {
+            if trip.stop_times.len() == 1 {
```

Comment:
The rule is

> A trip must visit more than one stop in stop_times.txt

a stop_times with 2 records for the same stop would pass with this code. WDYT about counting the distinct number of stops? This could be implemented by looking at the next trip as soon as we hit 2 different stop IDs.

Reply:
I guess I misinterpreted the rule.